# The commands to a bicycle model robot (v, phi)
float64 linear_velocity
float64 steering_rate